test_scripts/Capabilities/PersistingHMICapabilities/common.lua

```diff
@@ -118,22 +118,30 @@ function m.checkContentOfCapabilityCacheFile(pExpHmiCapabilities)
       for req, params in pairs(requests) do
         for _, param in ipairs(params) do
           local message = mod .. "." .. param
-          if param == "audioPassThruCapabilitiesList" then
-            if not utils.isTableEqual(cacheTable[mod].audioPassThruCapabilities, expHmiCapabilities[mod][req].params[param]) then
-              errorMessages = errorMessages ..
-                errorMessage(message, cacheTable[mod].audioPassThruCapabilities,
-                  expHmiCapabilities[mod][req].params[param])
-            end
-          else
-            if not cacheTable[mod][param] then
-              errorMessages = errorMessages ..
-                errorMessage(message, "does not exist", expHmiCapabilities[mod][req].params[param])
-            else
-              if not utils.isTableEqual(cacheTable[mod][param], expHmiCapabilities[mod][req].params[param]) then
+          if not cacheTable[mod][param] then
+            errorMessages = errorMessages ..
+              errorMessage(message, "does not exist", expHmiCapabilities[mod][req].params[param])
+          elseif param == "audioPassThruCapabilitiesList" then
+            if not utils.isTableEqual(cacheTable[mod].audioPassThruCapabilities,
+              expHmiCapabilities[mod][req].params[param]) then
```

Comment:
@VjKlepikov I would propose to create local variable to store value of `expHmiCapabilities[mod][req].params[param]` expression because it is calculated 3 times in short period of time.